pkg/utils/utils.go

```diff
@@ -51,9 +51,11 @@ func GetConn(ctx context.Context, socketPath string) (conn *grpc.ClientConn, err
 		return nil, err
 	}
 
+	//nolint:staticcheck // SA1019: grpc.DialContext is deprecated but maintains required blocking behavior
```

Comment:
can we open issues for these too

Reply:
created issue to track https://github.com/eraser-dev/eraser/issues/1188